packages/theme-chalk/src/upload.scss

```diff
@@ -254,20 +254,12 @@
       &:hover,
       &:focus {
         .#{bem('upload-list', 'item-status-label')} {
-          display: none;
```

Comment:
## Sugges

I think need `display: none`

because repeating mouse hover looks awkward.

<img width="988" alt="image" src="https://user-images.githubusercontent.com/27342882/219864709-55e95d05-ed8c-446d-a5e7-6291617702a0.png">

Reply:
Sorry for that, I change the code agian.